plugins/pulp_rpm/plugins/importers/yum/utils.py

```diff
@@ -102,6 +111,67 @@ def strip_ns(element, uri=None):
         strip_ns(child, uri)
 
 
+def fake_xml_element(repodata_snippet):
+    """
+    Wrap a snippet of xml in a fake element so it can be coerced to an ElementTree Element
+
+    :param repodata_snippet: Snippet of XML to be turn into an ElementTree Element
+    :type  repodata_snippet: str
+
+    :return: Parsed ElementTree Element containing the parsed repodata snippet
+    :rtype:  xml.etree.ElementTree.Element
+    """
+    register_namespace('rpm', constants.RPM_NAMESPACE)
+    try:
+        # make a guess at the encoding
+        codec = 'UTF-8'
+        repodata_snippet.encode(codec)
+    except UnicodeEncodeError:
+        # best second guess we have, and it will never fail due to the nature
+        # of the encoding.
+        codec = 'ISO-8859-1'
+    fake_xml = FAKE_XML % {'encoding': codec, 'xml': repodata_snippet,
+                           'namespace': constants.RPM_NAMESPACE}
+    # s/fromstring/phone_home/
```

Comment:
This comment has also been in place a long time, but I think it's a joke related to the use of ``ET`` below.